pilot/util/harvester.py

```diff
@@ -20,6 +20,11 @@
 logger = logging.getLogger(__name__)
 
 
+def dump(obj):
```

Comment:
Is this function used in some external script? Can you provide the documentation field?

Reply:
It is used for occasional debugging